plugins/coverage/info.js

```diff
@@ -19,10 +19,35 @@ module.exports = config => ({
             }
         },
         handler: (request, reply) => {
-            config.coveragePlugin
-                .getInfo(request.query)
-                .then(reply)
-                .catch(err => reply(boom.boomify(err)));
+            const { jobId, pipelineId, startTime, endTime, jobName, pipelineName, scope } = request.query;
+            const { jobFactory } = request.server.app;
+            const infoConfig = { jobId, pipelineId, startTime, endTime, jobName, pipelineName };
+
+            return Promise.resolve()
+                .then(() => {
+                    if (scope) {
+                        return { 'screwdriver.cd/coverageScope': request.query.scope };
+                    }
+                    if (!scope && jobId) {
```

Comment:
hopefully we can retire this end point once build end point fetches coverage info.

Reply:
yes, @klu909 should be working on that